addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -355,5 +356,13 @@ export class ColorPlugin extends Plugin {
         } else {
             element.style[mode] = color;
         }
+        if (mode === "backgroundColor" && isBlock(element)) {
+            if (color) {
+                const textColor = getComputedStyle(element).color;
+                element.style["color"] = textColor;
```

Comment:
unnecessary to assign this to a const to use it only once on the next line
```suggestion
                element.style["color"] = getComputedStyle(element).color;
```